.github/workflows/update-syscalls.yml

```diff
@@ -0,0 +1,44 @@
+name: Update Syscalls
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '* * * * 2'
+
+jobs:
+  update-syscalls:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        ref: drivers
```

Comment:
There is no `drivers` branch; you can just use master (and remove the `ref:` here)

Reply:
Thank you for pointing this out!